fhir-server-test/src/test/java/org/linuxforhealth/fhir/server/test/FHIRServerTestBase.java

```diff
@@ -547,13 +559,18 @@ protected boolean isSortingSupported() throws Exception {
     protected boolean isTransactionSupported() throws Exception {
         SystemRestfulInteraction transactionMode = null;
         CapabilityStatement conf = retrieveConformanceStatement();
-
+        boolean txnSupported = false;
         List<CapabilityStatement.Rest> restList = conf.getRest();
         if (restList != null) {
             CapabilityStatement.Rest rest = restList.get(0);
             if (rest != null) {
                 assertEquals(RestfulCapabilityMode.SERVER.getValue(), rest.getMode().getValue());
                 if (rest.getInteraction() != null) {
+                    for (org.linuxforhealth.fhir.model.resource.CapabilityStatement.Rest.Interaction interaction : rest.getInteraction()) {
+                        if (interaction.getCode().getValue().equals(SystemRestfulInteraction.TRANSACTION.getValue())) {
```

Comment:
just test code, but I think we should still prefer the enum flavor:
```suggestion
                        if (interaction.getCode().getValueAsEnum() == SystemRestfulInteraction.TRANSACTION) {
```

Reply:
i guess the current code already does the string comparison though (line 583), so maybe better to be consistent...

Reply:
This has been fixed as suggested. Removed redundant check in line 583 as well.